stdlib/public/BackDeployConcurrency/AsyncCompactMapSequence.swift

```diff
@@ -30,15 +30,15 @@ extension AsyncSequence {
   /// returns `nil` in this case, which `compactMap(_:)` omits from the
   /// transformed asynchronous sequence.
   ///
-  ///     let romanNumeralDict: [Int : String] =
+  ///     let romanNumeralDict: [Int: String] =
   ///         [1: "I", 2: "II", 3: "III", 5: "V"]
   ///         
   ///     let stream = Counter(howHigh: 5)
   ///         .compactMap { romanNumeralDict[$0] }
   ///     for await numeral in stream {
-  ///         print("\(numeral) ", terminator: " ")
+  ///         print(numeral, terminator: " ")
   ///     }
-  ///     // Prints: I  II  III  V
+  ///     // Prints "I II III V "
```

Comment:
If you think this doesn't need to be rear-spaced, just let me know.

Reply:
I think including the quotes is an improvement. It makes it easier to see that the output from `print` includes a space at the end.

Reply:
Good! I'll change `stdlib/public/Concurrency` files too and submit a new PR on the weekend. Thanks!